client/state/data-layer/utils.js

```diff
@@ -15,3 +19,11 @@ const doBypassDataLayer = {
 };
 
 export const bypassDataLayer = action => extendAction( action, doBypassDataLayer );
+
+/**
+ * Deeply converts keys from the specified object to camelCase notation.
+ *
+ * @param {Object} - object to convert
+ * @returns a new object with all keys converted
+ */
+export { humps as convertToCamelCase };
```

Comment:
I've created this _alias_ as `humps` is really cryptic, and also because converting to camelCase looks like common enough need for the data layer